### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by misaochan**
Update pull_request_template.md to include requirement for issue title and Javadocs

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
+{Please ensure that you have Javadocs written for all new methods and classes in your PR. Also, please test your PR at least once before submitting.}
```
Comment (maskaravivek) [inline PULL_REQUEST_TEMPLATE.md]: I don't think this is required for the PR template as long as the contributing guidelines have these instructions.

Comment (misaochan) [reply PULL_REQUEST_TEMPLATE.md]: Hmm, good point. However, does anyone actually read contributing.md before submitting their PR? :)

Comment (maskaravivek) [reply PULL_REQUEST_TEMPLATE.md]: Hmm, valid point. In that case we could add a link to the contributing guidelines in the PR template. 

However, PR templates is meant for just letting the contributor know what all things should be mentioned in the description for a good PR.

Comment (misaochan) [reply PULL_REQUEST_TEMPLATE.md]: True. Will modify it.

**Pull request merged**